sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 1.3.1 (2021-04-05)
+
+- Updated the default timeout of the first request of the IMDS MSI to three seconds to compensate for a situational issue caused by one of our dependencies in which the timeout would end before the first request was sent.
```

Comment:
What is a situational issue?

Any reason to not be transparent about which dependency is causing the issue?

Also, since this affects only certain environments, only certain set of users would be interested in the fix. We should ensure that they get to identify that this fix is important to them. We can do that by being transparent about which environment have we seen this problem in

```suggestion
- Updated the default timeout of the first request of the IMDS MSI from half a second to three seconds to compensate for the slowness caused by `node-fetch` for initial requests in certain environments like kubernetes pods.
```